js/private/js_run_devserver.mjs

```
@@ -243,6 +272,7 @@ async function syncRecursive(src, dst, sandbox, writePerm) {
         }
     } catch (e) {
         console.error(e)
+        console.trace()
```

What does this output?

---

Ah yes, without this we were having a hard time telling where errors were coming from because the exception was swallowed, logged and then program exited. So in our logs we would just see 

```
...
ENOENT, open /whatever
Process died
```

[`console.trace()`](https://developer.mozilla.org/en-US/docs/Web/API/console/trace_static) logs a stack trace alongside this error so you can see which line of code originally threw the exception